etc/scripts/test-integ-mysql.sh

```diff
@@ -52,8 +52,12 @@ mvn ${MAVEN_ARGS} -f ${WS_DIR}/pom.xml \
 
 # Run tests in Java VM application
 (cd tests/integration/jpa && \
-  mvn ${MAVEN_ARGS} clean verify -Dmysql)
+  mvn ${MAVEN_ARGS} clean verify \
+      -Dmaven.test.failure.ignore=true -Dmysql \
```

Comment:
This is wrong - this PR is for `psql` not for `mysql`?

Reply:
There was -Dmaven.test.failure.ignore=true  missing so I added it into both scripts. They are not being used now and we'll do another review after Romain prepares parallel jobs setup for all those native image integration tests.